src/test/java/com/google/cloud/resourcemanager/ResourceManagerImplTest.java

```diff
@@ -456,4 +456,17 @@ public void testRuntimeException() {
       assertEquals(exceptionMessage, exception.getCause().getMessage());
     }
   }
+
+  @Test
+  public void testReplaceOrgPolicy() {
+    try {
+      Policy policy = RESOURCE_MANAGER.replaceOrgPolicy("organizations/12345", POLICY);
+      assertEquals(POLICY.getBindings(), policy.getBindings());
+      assertNotNull(policy.getEtag());
```

Comment:
what method throws the exception? There's at least one assert here that isn't evaluated.

Reply:
done